addons/point_of_sale/static/src/js/Popups/AlertPopup.js

```diff
@@ -0,0 +1,19 @@
+odoo.define('point_of_sale.AlertPopup', function(require) {
+    'use strict';
+
+    const AbstractAwaitablePopup = require('point_of_sale.AbstractAwaitablePopup');
+    const Registries = require('point_of_sale.Registries');
+    const { _lt } = require('@web/core/l10n/translation');
+
+    class AlertPopup extends AbstractAwaitablePopup {}
```

Comment:
Can't you use an Error Popup (or improve it) as it really look similar

Reply:
You know what, i'm gonna use the toast notification that has recently been implemented. It serves the same purpose